Source/Particles/Pusher/UpdateMomentumBorisWithRadiationReaction.H

```diff
@@ -0,0 +1,82 @@
+
+#ifndef WARPX_PARTICLES_PUSHER_UPDATEMOMENTUM_BORIS_WITHRR_H_
+#define WARPX_PARTICLES_PUSHER_UPDATEMOMENTUM_BORIS_WITHRR_H_
+
+#include <AMReX_REAL.H>
+#include <UpdateMomentumBoris.H>
+
+/* \brief Push the particle's positions over one timestep,
```

Comment:
Instead of 
```
/* \brief Push the particle's positions over one timestep,
```
I think Doxygen syntax requires
```
/** \brief Push the particle's positions over one timestep,
```
or
```
/**
 * \brief Push the particle's positions over one timestep,
```

Reply:
It _should_ work now...